src/components/edit-profile/container.test.tsx

```diff
@@ -14,6 +14,7 @@ describe('Container', () => {
       currentProfileImage: 'profile.jpg',
       currentPrimaryZID: '0://john:doe',
       ownedZIDs: [],
+      loading: false,
```

Comment:
Just a nit, loading seems a bit vague. I'd have to dig into the saga to check what is actually loading if I didn't have the context.

Reply:
most saga's have this `loading` prop to indicate "loading" state. If not clear, we can also use `EditProfile.State.IS_LOADING` (i.e use the `State` enum)

Reply:
I think Dom's point is that this flag is not indicating that we're loading "the profile". It's very specific in that we're loading the zids. What if we had different data to load that could be rendered early, for example.